backend/open_webui/config.py

```
@@ -2,29 +2,31 @@
 import logging
 import os
 import shutil
+import base64
```

Ruff (0.8.2)

`base64` imported but unused

Remove unused import: `base64`

(F401)